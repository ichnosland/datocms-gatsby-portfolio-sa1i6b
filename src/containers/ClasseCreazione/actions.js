/*
 *
 * ClasseCreazione actions
 *
 */

import {
  CLASSE_CREAZIONE_SCUOLE_ATTIVE_FETCH,
  CLASSE_CREAZIONE_SCUOLE_ATTIVE_SET,
  CLASSE_CREAZIONE_GEO_PROVINCIA_FETCH,
  CLASSE_CREAZIONE_GEO_SET,
  CLASSE_CREAZIONE_GEO_COMUNI_FETCH,
  CLASSE_CREAZIONE_GEO_COMUNI_SET,
  CLASSE_CREAZIONE_DATA_SET,
  CLASSE_CREAZIONE_DATA_POST,
  CLASSE_CREAZIONE_RESET,
  CLASSE_CREAZIONE_SPINNER_SET,
  CLASSE_CREAZIONE_FEEDBACK_SET,
  CLASSE_CREAZIONE_FEEDBACK_RESET,
  CLASSE_CREAZIONE_GEO_SCUOLE_SET,
} from './constants';


export const classeCreazioneScuoleAttiveFetch = () => ({
  type: CLASSE_CREAZIONE_SCUOLE_ATTIVE_FETCH,
});

export const classeCreazioneScuoleAttiveSet = (payload) => ({
  type: CLASSE_CREAZIONE_SCUOLE_ATTIVE_SET,
  payload,
});

export const classeCreazioneGeoProvinciaFetch = () => ({
  type: CLASSE_CREAZIONE_GEO_PROVINCIA_FETCH,
});

export const classeCreazioneGeoSet = (payload) => ({
  type: CLASSE_CREAZIONE_GEO_SET,
  payload,
});

export const classeCreazioneGeoComuneFetch = (sigla) => ({
  type: CLASSE_CREAZIONE_GEO_COMUNI_FETCH,
  sigla,
});

export const classeCreazioneGeoComuneSet = (payload) => ({
  type: CLASSE_CREAZIONE_GEO_COMUNI_SET,
  payload,
});

export const classeCreazioneGeoScuoleSet = (payload) => ({
  type: CLASSE_CREAZIONE_GEO_SCUOLE_SET,
  payload,
});

export const classeCreazioneDataSet = (payload) => ({
  type: CLASSE_CREAZIONE_DATA_SET,
  payload,
});

export const classeCreazioneDataPost = (payload, history) => ({
  type: CLASSE_CREAZIONE_DATA_POST,
  payload,
  history,
});

export const classeCreazioneReset = () => ({
  type: CLASSE_CREAZIONE_RESET,
});

export const classeCreazioneSpinnerSet = (enable) => ({
  type: CLASSE_CREAZIONE_SPINNER_SET,
  enable,
});

export const classeCreazioneFeedbackSet = (enable, tipologia, messaggio) => ({
  type: CLASSE_CREAZIONE_FEEDBACK_SET,
  enable,
  tipologia,
  messaggio,
});

export const classeCreazioneFeedbackReset = () => ({
  type: CLASSE_CREAZIONE_FEEDBACK_RESET,
});
