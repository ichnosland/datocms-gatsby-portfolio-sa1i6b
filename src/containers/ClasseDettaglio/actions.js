/*
 *
 * ClasseDettaglio actions
 *
 */

import {
  CLASSE_DETTAGLIO_ESPELLI,
  CLASSE_DETTAGLIO_RESET,
  CLASSE_DETTAGLIO_DATA_FETCH,
  CLASSE_DETTAGLIO_CORSO_FETCH,
  CLASSE_DETTAGLIO_CONTENUTO_SET,
  CLASSE_DETTAGLIO_DISPLAY_SET,
  CLASSE_DETTAGLIO_DISPLAY_SORT,
  CLASSE_DETTAGLIO_FEEDBACK_SET,
  CLASSE_DETTAGLIO_FEEDBACK_RESET,
  CLASSE_DETTAGLIO_SPINNER_SET,
} from './constants';

export const classeDettaglioEspelli = (payload) => ({
  type: CLASSE_DETTAGLIO_ESPELLI,
  payload,
});

export const classeDettaglioReset = () => ({
  type: CLASSE_DETTAGLIO_RESET,
});

export const classeDettaglioCorsoFetch = (payload) => ({
  type: CLASSE_DETTAGLIO_CORSO_FETCH,
  payload,
});

export const classeDettaglioDataFetch = (payload) => ({
  type: CLASSE_DETTAGLIO_DATA_FETCH,
  payload,
});

export const classeDettaglioContenutoSet = (payload) => ({
  type: CLASSE_DETTAGLIO_CONTENUTO_SET,
  payload,
});

export const classeDettaglioDisplaySort = (payloadData, sortingData) => ({
  type: CLASSE_DETTAGLIO_DISPLAY_SORT,
  payloadData,
  sortingData,
});

export const classeDettaglioDisplaySet = (payload) => ({
  type: CLASSE_DETTAGLIO_DISPLAY_SET,
  payload,
});

export const classeDettaglioFeedbackSet = (hasFeedback, tipologia, messaggio) => ({
  type: CLASSE_DETTAGLIO_FEEDBACK_SET,
  hasFeedback,
  tipologia,
  messaggio,
});

export const classeDettaglioFeedbackReset = () => ({
  type: CLASSE_DETTAGLIO_FEEDBACK_RESET,
});

export const classeDettaglioSpinnerSet = (enable) => ({
  type: CLASSE_DETTAGLIO_SPINNER_SET,
  enable,
});
