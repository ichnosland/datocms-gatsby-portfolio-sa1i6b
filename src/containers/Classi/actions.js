/*
 *
 * Classi actions
 *
 */

import {
  CLASSI_DATA_FETCH,
  CLASSI_DATA_SET,
  CLASSI_DATA_RESET,
  CLASSI_FEEDBACK_RESET,
  CLASSI_FEEDBACK_SET,
  CLASSI_SPINNER_SET,
} from './constants';

export const classiDataFetch = (idDisciplina) => ({
  type: CLASSI_DATA_FETCH,
  idDisciplina,
});

export const classiDataSet = (payload) => ({
  type: CLASSI_DATA_SET,
  payload,
});

export const classiDataReset = () => ({
  type: CLASSI_DATA_RESET,
});

export const classiFeedbackReset = () => ({
  type: CLASSI_FEEDBACK_RESET,
});

export const classiFeedbackSet = (enable = true, message = '', errorData = {}) => ({
  type: CLASSI_FEEDBACK_SET,
  enable,
  message,
  errorData,
});

export const classiSpinnerSet = (enable) => ({
  type: CLASSI_SPINNER_SET,
  enable,
});
