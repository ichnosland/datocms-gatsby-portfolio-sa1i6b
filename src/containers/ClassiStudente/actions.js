/*
 *
 * ClassiStudente actions
 *
 */

import {
  CLASSI_STUDENTE_FETCH,
  CLASSI_STUDENTE_SET,
  CLASSI_STUDENTE_RESET,
  CLASSI_STUDENTE_ISCRIVITI_POST,
  CLASSI_STUDENTE_SPINNER_SET,
  CLASSI_STUDENTE_FEEDBACK_SET,
  CLASSI_STUDENTE_FEEDBACK_RESET,
} from './constants';

export const classiStudenteFetch = (disciplinaId) => ({
  type: CLASSI_STUDENTE_FETCH,
  disciplinaId,
});

export const classiStudenteSet = (payload) => ({
  type: CLASSI_STUDENTE_SET,
  payload,
});

export const classiStudenteReset = () => ({
  type: CLASSI_STUDENTE_RESET,
});

export const classiStudenteIscrivitiPost = (codiceCorso, configuration) => ({
  type: CLASSI_STUDENTE_ISCRIVITI_POST,
  codiceCorso,
  configuration,
});

export const classiStudenteSpinnerSet = (enable) => ({
  type: CLASSI_STUDENTE_SPINNER_SET,
  enable,
});

export const classiStudenteFeedbackSet = (enable, tipologia, messaggio) => ({
  type: CLASSI_STUDENTE_FEEDBACK_SET,
  enable,
  tipologia,
  messaggio,
});

export const classiStudenteFeedbackReset = () => ({
  type: CLASSI_STUDENTE_FEEDBACK_RESET,
});
