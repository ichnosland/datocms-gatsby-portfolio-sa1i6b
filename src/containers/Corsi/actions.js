/*
 *
 * Corsi actions
 *
 */

import {
  CORSI_RESET,
  CORSI_ERROR_SET,
  CORSI_ERROR_RESET,
  CORSI_SPINNER_SET,
  CORSI_LIST_INITIALIZE,
  CORSI_LIST_DOCENTE_FETCH,
  CORSI_LIST_STUDENTE_FETCH,
  CORSI_LIST_SET,
  CORSI_CORSO_DETAIL_DOCENTE_TRIGGER,
  CORSI_CORSO_DETAIL_DOCENTE_SET,
} from './constants';

export const corsiReset = () => ({
  type: CORSI_RESET,
});

export const corsiSpinnerSet = (enable) => ({
  type: CORSI_SPINNER_SET,
  enable,
});

export const corsiErrorSet = (enable = true, message = '', errorData = {}) => ({
  type: CORSI_ERROR_SET,
  enable,
  message,
  errorData,
});

export const corsiErrorReset = () => ({
  type: CORSI_ERROR_RESET,
});

export const corsiListInitialize = (configuration, isDocente, userId) => ({
  type: CORSI_LIST_INITIALIZE,
  configuration,
  isDocente,
  userId,
});

export const corsiListDocenteFetch = (idDisciplina) => ({
  type: CORSI_LIST_DOCENTE_FETCH,
  idDisciplina,
});

export const corsiListStudenteFetch = (idDisciplina) => ({
  type: CORSI_LIST_STUDENTE_FETCH,
  idDisciplina,
});

export const corsiListSet = (payload) => ({
  type: CORSI_LIST_SET,
  payload,
});

export const corsiCorsoSelectedDocenteTrigger = (idCorso, configuration, isDocente, userId) => ({
  type: CORSI_CORSO_DETAIL_DOCENTE_TRIGGER,
  idCorso,
  configuration,
  isDocente,
  userId,
});

export const corsiCorsoDetailDocenteSet = (payload) => ({
  type: CORSI_CORSO_DETAIL_DOCENTE_SET,
  payload,
});
