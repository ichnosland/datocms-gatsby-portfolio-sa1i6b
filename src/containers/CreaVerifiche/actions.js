/*
 *
 * CreaVerifiche actions
 *
 */

import {
  CREA_VERIFICHE_FETCH,
  CREA_VERIFICHE_FETCH_ERROR,
  CREA_VERIFICHE_PREVIEW_ESERCIZIO_FETCH,
  CREA_VERIFICHE_PREVIEW_ESERCIZIO_FETCH_ERROR,
  CREA_VERIFICHE_SET,
  CREA_VERIFICHE_SPINNER_SET,
  CREA_VERIFICHE_POST,
  CREA_VERIFICHE_POST_ERROR,
  CREA_VERIFICHE_VERIFICA_SET,
  CREA_VERIFICHE_VERIFICA_ESERCIZIO_ADD,
  CREA_VERIFICHE_VERIFICA_ESERCIZIO_REMOVE,
  CREA_VERIFICHE_VERIFICA_RIEPILOGO_VIEW,
} from './constants';

export const creaverificheVerificaPostError = (messaggio = 'Impossibile inviare i dati') => ({
  type: CREA_VERIFICHE_POST_ERROR,
  messaggio,
});

export const creaverificheVerificaPost = (payload) => ({
  type: CREA_VERIFICHE_POST,
  payload,
});

export const creaverificheVerificaEserciziFetch = (prerequisito) => ({
  type: CREA_VERIFICHE_FETCH,
  prerequisito,
});

export const creaverificheVerificaEserciziFetchError = (messaggio = 'Impossibile caricare l\'elenco degli esercizi disponibili') => ({
  type: CREA_VERIFICHE_FETCH_ERROR,
  messaggio,
});

export const creaverificheVerificaEsercizioPreviewFetch = (esercizioId) => ({
  type: CREA_VERIFICHE_PREVIEW_ESERCIZIO_FETCH,
  esercizioId,
});

export const creaverificheVerificaEsercizioPreviewFetchError = (
  messaggio = 'Impossibile caricare l\'elenco degli step per questo esercizio'
) => ({
  type: CREA_VERIFICHE_PREVIEW_ESERCIZIO_FETCH_ERROR,
  messaggio,
});

export const creaverificheVerificaEserciziSet = (payload) => ({
  type: CREA_VERIFICHE_SET,
  payload,
});

export const creaverificheVerificaEserciziSpinnerSet = (enabled) => ({
  type: CREA_VERIFICHE_SPINNER_SET,
  enabled,
});

export const creaverificheVerificaSet = (payload) => ({
  type: CREA_VERIFICHE_VERIFICA_SET,
  payload,
});

export const creaverificheVerificaEsercizioAdd = (step) => ({
  type: CREA_VERIFICHE_VERIFICA_ESERCIZIO_ADD,
  step,
});

export const creaverificheVerificaEsercizioRemove = (step) => ({
  type: CREA_VERIFICHE_VERIFICA_ESERCIZIO_REMOVE,
  step,
});

export const creaverificheVerificaRiepilogoView = (enabled) => ({
  type: CREA_VERIFICHE_VERIFICA_RIEPILOGO_VIEW,
  enabled,
});
