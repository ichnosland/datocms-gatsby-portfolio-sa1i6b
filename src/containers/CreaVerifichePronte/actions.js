/*
 *
 * CreaVerifichePronte actions
 *
 */

import {
  CREA_VERIFICHE_PRONTE_FETCH,
  CREA_VERIFICHE_PRONTE_FETCH_ERROR,
  CREA_VERIFICHE_PRONTE_SET,
  CREA_VERIFICHE_PRONTE_SPINNER_SET,
  CREA_VERIFICHE_PRONTE_FETCH_ESERCIZI,
} from './constants';

export const creaverificheVerifichePronteFetch = (disciplinaId) => ({
  type: CREA_VERIFICHE_PRONTE_FETCH,
  disciplinaId,
});

export const creaverificheVerifichePronteFetchError = (messaggio = 'Impossibile caricare l\'elenco delle verifiche pronte') => ({
  type: CREA_VERIFICHE_PRONTE_FETCH_ERROR,
  messaggio,
});

export const creaverificheVerifichePronteSet = (payload) => ({
  type: CREA_VERIFICHE_PRONTE_SET,
  payload,
});

export const creaverificheVerifichePronteSpinnerSet = (enabled) => ({
  type: CREA_VERIFICHE_PRONTE_SPINNER_SET,
  enabled,
});

export const creaverificheVerifichePronteEserciziFetch = (payload) => ({
  type: CREA_VERIFICHE_PRONTE_FETCH_ESERCIZI,
  payload,
});
