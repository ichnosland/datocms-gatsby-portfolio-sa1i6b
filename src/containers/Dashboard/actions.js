import {
  DASHBOARD_UNITA_FILTRA,
  DASHBOARD_LIVELLI_SET,
  DASHBOARD_LIVELLI_FETCH,
  DASHBOARD_LIVELLI_FETCH_ERROR,
  DASHBOARD_SPINNER_SET,
  DASHBOARD_LIVELLI_RESET,
  DASHBOARD_MENUEGG_SET,
  DASHBOARD_SEARCHACTIVE_SET,
  DASHBOARD_LIVELLI_ISLOADING_SET,
} from './constants';

export function livelliFetch(configuration, isDocente = false, corsoId = undefined) {
  return {
    type: DASHBOARD_LIVELLI_FETCH,
    configuration,
    isDocente,
    corsoId,
  };
}

export function livelliSet(payload) {
  return {
    type: DASHBOARD_LIVELLI_SET,
    payload,
  };
}

export function livelliReset() {
  return {
    type: DASHBOARD_LIVELLI_RESET,
  };
}

export function livelliFetchError(message = 'Errore caricamento') {
  return {
    type: DASHBOARD_LIVELLI_FETCH_ERROR,
    message,
  };
}

export function livelliIsLoadingSet(value) {
  return {
    type: DASHBOARD_LIVELLI_ISLOADING_SET,
    value,
  };
}

export function spinnerSet(payload) {
  return {
    type: DASHBOARD_SPINNER_SET,
    payload,
  };
}

export function unitaFiltra(parolaChiave) {
  return {
    type: DASHBOARD_UNITA_FILTRA,
    filtro: parolaChiave,
  };
}

export const dashboardMenuEggSet = (value) => ({
  type: DASHBOARD_MENUEGG_SET,
  value,
});

export const dashboardSearchActiveSet = (value) => ({
  type: DASHBOARD_SEARCHACTIVE_SET,
  value,
});
