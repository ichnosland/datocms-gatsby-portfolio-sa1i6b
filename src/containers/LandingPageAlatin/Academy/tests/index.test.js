import React from 'react';
import { shallow } from 'enzyme';
import { connect } from 'react-redux';
import { compose } from 'redux';

import configureStore from 'configureStore';
import injectReducer from 'utils/injectReducer';
import reducerCurryer from 'common/applications/reducer';
import ladingPageReducer from 'containers/LandingPage/reducer';

import { StickyWrap } from 'components/StickyFooter';
import { landingPageToggleMenu } from 'containers/LandingPage/actions';
import AcademyComposed, { Academy } from '../index';

const configuration = {
  disciplinaId: 21,
  product: 'landingalatin',
  blocco: false,
  titoloApplicazione: 'Alatin Academy',
  homePage: '/',
  hasClassi: false,
};

const mockProps = {
  configuration,
  landingPage: {
    isMenuOpened: true,
  },
  onlandingPageToggleMenu: () => {},
  match: {
    path: '/path',
  },
};

describe('<Academy />', () => {
  it('should render a StickyWrap', () => {
    const renderedComponent = shallow(
      <Academy {...mockProps} />
    );
    expect(renderedComponent.type()).toEqual(StickyWrap);
    expect(renderedComponent).toMatchSnapshot();
  });
});

describe('<AcademyComposed />', () => {
  const withReducerConfiguration = injectReducer({ key: 'configuration', reducer: reducerCurryer(configuration) });
  const withLandingConfiguration = injectReducer({ key: 'landingPage', reducer: ladingPageReducer });
  const ConfigurationWrapper = compose(
    withReducerConfiguration,
    withLandingConfiguration,
    connect()
  )(AcademyComposed);
  const mockStore = configureStore({}, {});
  mockStore.dispatch = jest.fn();

  it('onModalSetData deve chiamare modalSetData', () => {
    const renderedComponent = shallow(
      <ConfigurationWrapper match={mockProps.match} />, { context: { store: mockStore } }
    ).dive().dive().dive();

    const receivedProps = renderedComponent.props();
    expect(receivedProps.configuration).toEqual(
      configuration
    );

    expect(mockStore.dispatch).not.toHaveBeenCalledWith(
      landingPageToggleMenu(true)
    );
    receivedProps.onlandingPageToggleMenu(true);
    expect(mockStore.dispatch).toHaveBeenCalledWith(
      landingPageToggleMenu(true)
    );
  });
});
