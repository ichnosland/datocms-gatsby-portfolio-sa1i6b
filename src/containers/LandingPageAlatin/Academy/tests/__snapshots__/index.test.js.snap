// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Academy /> should render a StickyWrap 1`] = `
<StickyFooter__StickyWrap>
  <StickyFooter__StickyTop>
    <ComponentWithScrollPosition />
    <Section
      bgColor="#00abe5"
      first={true}
    >
      <LandingCard
        color="#fff"
        flexorder={2}
        src="IMAGE_MOCK"
      >
        <Heading__H1
          color="#fff"
          margin="0"
        >
          Alatin Academy
        </Heading__H1>
        <p>
          Alatin è un corso di lingua basato sull’apprendimento per padronanza o 
          <strong>
            “Mastery Learning”
          </strong>
          . Il docente non assegna un compito, bensì un obiettivo comune a tutta la classe, ad esempio “3ª declinazione”. Gli studenti sono tenuti a completare tutte le lezioni pratiche comprese nella 3ª declinazione, ma la mole di esercizi che toccherà loro varierà in base al numero degli errori compiuti nel corso delle lezioni.
        </p>
        <p>
          In ogni lezione pratica, lo studente è tenuto a raggiungere 12 risposte corrette; ogni volta che sbaglia, gli viene proposto un nuovo esercizio; in coda, per chiudere la lezione, dovrà correggere tutte le domande che ha sbagliato.
        </p>
        <p>
          Completata una lezione lo studente sblocca e accede alle lezioni successive, con domande ed esercizi via via più difficili, fino al raggiungimento dell’obiettivo.
        </p>
      </LandingCard>
    </Section>
    <Section
      bgColor="#F2F2F2"
    >
      <Container>
        <Heading__H2
          color="#00abe5"
          margin="0"
        >
          Struttura del corso
        </Heading__H2>
        <p>
          Alatin è organizzato in 
          <strong>
            180 unità
          </strong>
           (obiettivi di apprendimento), scandite in 
          <strong>
            40 missioni
          </strong>
          , raggruppate a loro volta in 
          <strong>
            14 livelli
          </strong>
          . Ogni unità (es. «1ª declinazione») contiene un numero variabile di 
          <strong>
            lezioni pratiche
          </strong>
          , in cui lo studente affronta esercizi di morfologia, morfosintassi e sintassi a difficoltà crescente, dall’analisi dei verbi alla traduzione di frasi.
        </p>
        <RespImg__DesktopImage
          src="IMAGE_MOCK"
        />
        <RespImg__MobileImage
          src="IMAGE_MOCK"
        />
        <Heading__H4
          color="#000"
        >
          Le lezioni pratiche
        </Heading__H4>
        <p>
          Ogni lezione pratica di 
          <em>
            traduzione
          </em>
           è scandita da una continua dialettica fra le lingue: si traduce cioè non solo dal latino all’italiano ma, in egual misura, dall’italiano al latino.
        </p>
        <p>
          Ciascuna missione è organizzata in modo da consentire allo studente un’acquisizione mirata e progressiva del lessico di base. Il vocabolario è in linea, dunque lo studente trova il lemma della parola direttamente sul testo. Se la parola è focus, cioè si tratta di una parola-chiave per il tipo di esercizio richiesto (nome, verbo, aggettivo), è corredata dalla rispettiva tabella morfologica che ne riporta il quadro dei casi o della coniugazione.
        </p>
        <RespImg__DesktopImage
          src="IMAGE_MOCK"
        />
        <RespImg__MobileImage
          src="IMAGE_MOCK"
        />
        <p>
          Oltre al lessico, gli studenti trovano in Alatin anche la teoria. Ciascuna unità, infatti, è dotata di una breve lezione teorica di supporto all’argomento trattato, in linea con quanto avviene nei laboratori tradizionali.
        </p>
        <RespImg__DesktopImage
          src="IMAGE_MOCK"
        />
        <RespImg__MobileImage
          src="IMAGE_MOCK"
        />
        <Heading__H4
          color="#000"
        >
          La valutazione degli obiettivi
        </Heading__H4>
        <p>
           Per ogni unità il docente ha a disposizione un tasto “Andamento” che gli consente di monitorare in tempo reale, su 3 indici, il comportamento della classe rispetto al lavoro assegnato. I 3 indici rappresentati sono: 
        </p>
        <ul>
           
          <li>
            grado di completamento dell’obiettivo
          </li>
           
          <li>
            numero di errori commessi
          </li>
           
          <li>
            punteggio esperienza complessivo (XP) raggiunto in 
            <strong>
              tutti
            </strong>
             gli obiettivi
          </li>
           
        </ul>
        <RespImg__DesktopImage
          src="IMAGE_MOCK"
        />
        <RespImg__MobileImage
          src="IMAGE_MOCK"
        />
        <Heading__H4
          color="#000"
        >
          Le verifiche e le versioni
        </Heading__H4>
        <p>
          In ogni missione il docente dispone di 3 o più 
          <strong>
            versioni
          </strong>
          , che può decidere se assegnare tutte o in parte a scuola, a titolo di compito in classe, o a casa. Le versioni sono valutate in 
          <strong>
            decimi
          </strong>
          .
          <br />
           Ciascuna versione è scandita per periodi, che lo studente deve prima analizzare e poi tradurre passo a passo. La valutazione tiene conto in egual misura degli step di analisi e di quelli di traduzione.
        </p>
        <p>
           Per ogni livello il docente può creare delle 
          <strong>
            verifiche sommative
          </strong>
           personalizzabili sulla base delle unità assegnate, dalla morfologia alla traduzione, basate rigorosamente sul medesimo lessico che i ragazzi trovano nel lavoro a casa.
          <br />
           Le verifiche tuttavia, a differenza degli obiettivi assegnati per casa, non hanno feedback immediato e non hanno i traducenti in linea: solo chi si esercita costantemente a casa riuscirà a prendere un buon voto nelle verifiche!
        </p>
        <RespImg__DesktopImage
          src="IMAGE_MOCK"
        />
        <RespImg__MobileImage
          src="IMAGE_MOCK"
        />
        <Heading__H4
          color="#000"
        >
          Perché provare Alatin?
        </Heading__H4>
        <p>
          Da sempre il Mastery Learning si scontra con le difficoltà dei docenti di compensare gli squilibri fisiologici di una classe con compiti mirati al singolo studente. Alatin è pensato per offrire al docente la possibilità di superare questa impasse.
          <br />
          Monitorando minuziosamente tutto il lavoro di ogni studente Alatin è in grado di ponderare automaticamente il suo carico di lavoro, fino a fargli raggiungere il traguardo fissato. Al tempo stesso, Alatin riporta al docente le difficoltà incontrate dalla classe nel raggiungimento dei singoli traguardi, offrendo ottimi spunti anche per la lezione frontale.
          <br />
          È possibile utilizzare le lezioni di Alatin sia come attività di pratica e consolidamento dell’argomento appena trattato in classe, indicando quest’ultimo come traguardo da superare per la volta successiva, sia come prologo dell’argomento non ancora affrontato in classe, proprio a partire dagli errori più frequenti che la classe ha commesso a casa.
          <br />
          E ciò che vale per la fase lineare della programmazione tradizionale, vale a maggior ragione per i momenti puntuali di potenziamento e di recupero.
        </p>
      </Container>
    </Section>
    <Section
      bgColor="#F2F2F2"
      padding="0 50px 50px"
      paddingmobile="0 20px 30px"
    >
      <Container
        align="center"
      >
        <ButtonLink__ActionButtonLink
          bgcolor="#fe9d3e"
          sectionbutton={1}
          shadow={1}
          to="/contatti"
        >
          Scrivici per saperne di più
        </ButtonLink__ActionButtonLink>
      </Container>
    </Section>
  </StickyFooter__StickyTop>
  <StickyFooter__StickyBottom>
    <ZigZagBorder />
    <Footer />
  </StickyFooter__StickyBottom>
</StickyFooter__StickyWrap>
`;
